src/nvim/eval.c

```diff
@@ -12882,6 +12885,7 @@ static void f_rpcstart(typval_T *argvars, typval_T *rettv)
   uint64_t channel_id = channel_from_job(argv);
 
   if (!channel_id) {
+on_error:
```

Comment:
`channel_id` is not initialized when we jump to this point and use it later as return value.